### ISSUE TITLE
Fix slack timestamps

### ISSUE THREAD

**Issue by hongee** [daily-bruin/meow#53]
The time meow is reporting on Slack looks like a UTC time instead of the set timezone. Needs a fix.

**Comment by brmi** [comment]
So I changed the code in the view.py file so the dates displayed are like below when you first load the page. I tried to add a post and I got the send error "NoneError: Daily Bruin 2017-10-18 07:46:23.662486 -- Would have sent more than 20 minutes late." So, I think the problem is in models.py maybe? Anyway, just an update. Gonna look at models.py tomorrow :) 

<img width="1063" alt="screen shot 2017-10-18 at 12 53 27 am" src="https://user-images.githubusercontent.com/7662531/31706755-e816b826-b39e-11e7-97b5-3996ebc1ada7.png">
<img width="1028" alt="screen shot 2017-10-18 at 12 53 33 am" src="https://user-images.githubusercontent.com/7662531/31706762-edd6012c-b39e-11e7-838b-fb0bf4e4b9db.png">
<img width="1013" alt="screen shot 2017-10-18 at 12 53 37 am" src="https://user-images.githubusercontent.com/7662531/31706769-f2ac1ae2-b39e-11e7-9fa5-433884a6720a.png">
<img width="991" alt="screen shot 2017-10-18 at 12 52 23 am" src="https://user-images.githubusercontent.com/7662531/31706773-f71ad604-b39e-11e7-8ac5-c83f7b88b9f8.png">
